sea-query-driver/Cargo.toml

```diff
@@ -33,6 +33,8 @@ with-time = []
 with-ipnetwork = []
 with-mac_address = []
 postgres-array = []
+postgres-cidr = []
+postgres-eui48 = []
```

Comment:
Hey @ikrivosheev, I think we also need to match the `Value`s of `cidr` and `eui48`? Just like what we did to `Value::Array`:
- https://github.com/SeaQL/sea-query/blob/190eb5041e44173a560bb252f1dd4dd6d07fc77b/sea-query-driver/src/sqlx_postgres.rs#L59-L63

Reply:
@billy1624 hello! In each driver we have:

```
_ => unimplemented!(),
```